.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2.1
 executors:
   golang:
     docker:
-      - image: cimg/go:1.19
```

Comment:
We are still on [Go 1.19](https://github.com/CosmWasm/wasmd/blob/main/go.mod#L3) . Let's have the Go upgrade in a separate PR. We have https://github.com/CosmWasm/wasmd/issues/1385 for the next release

Reply:
sure thing